tests/unit/test_charm.py

```diff
@@ -215,7 +236,7 @@ def test_given_service_template_when_install_then_srsenb_service_file_is_rendere
             "Type=simple\n"
             "Restart=always\n"
             "RestartSec=1\n"
-            "ExecStart=/build/srsue/src/srsue --usim.algo=milenage --nas.apn=oai.ipv4 --rf.device_name=zmq --rf.device_args=tx_port=tcp://*:2001,rx_port=tcp://localhost:2000,id=ue,base_srate=23.04e6 /config/ue.conf\n"  # noqa: E501, W505
+            "ExecStart=/build/srsue/src/srsue --usim.imsi=whatever imsi --usim.k=whatever k --usim.opc=whatever opc --usim.algo=milenage --nas.apn=oai.ipv4 --rf.device_name=zmq --rf.device_args=tx_port=tcp://*:2001,rx_port=tcp://localhost:2000,id=ue,base_srate=23.04e6 /config/ue.conf\n"  # noqa: E501, W505
```

Comment:
Why are there spaces in between? `--usim.imsi=whatever imsi --usim.k=whatever k --usim.opc=whatever opc`
Am I missing something?

Reply:
good catch :D fixed